templates/rsyslog.conf.erb

```diff
@@ -55,23 +55,9 @@ $ActionSendStreamDriverPermittedPeer <%= @permitted_peer %>
 $RuleSet local
 
 <% end -%>
-# Log all kernel messages to the console.
-# Logging much else clutters up the screen.
-#kern.*                                                 /dev/console
-kern.*                                                  <%= @kernel_target %>
-
-# Log anything (except mail) of level info or higher.
-# Don't log private authentication messages!
-*.info;mail.none;authpriv.none;cron.none                /var/log/messages
-
-# The authpriv file has restricted access.
-authpriv.*                                              /var/log/secure
-
-# Log all the mail messages in one place.
-mail.*                                                  -/var/log/maillog
-
-# Log cron stuff
-cron.*                                                  /var/log/cron
+<% @log_entries_real.each do |log_entry| -%>
+<%= log_entry %>
+<% end -%>
 
 # Everybody gets emergency messages
```

Comment:
What do you think about putting this logic in the manifest and then using a variable for the *.emerg target and handling that in the array, similar to how $kernel_target is handled?